go/sqldb/sql_error.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package sqldb
+
+import "fmt"
+
+// SqlError is the error structure returned from calling a db library function
+type SqlError struct {
```

Comment:
Since SqlError is essentially "an error that comes from the sqldb package", the no-stutter rule would suggest that this should be called just Error, so it is written in other packages as sqldb.Error.